### ISSUE TITLE
 training Auxiliary classifier of Inception V3

### ISSUE THREAD

**Issue by bis-carbon** [mikevoets/jama16-retina-replication#10]
Hello, 
Since you are training the full network did you consider training the whole Inception-v3 along with the auxiliary classifier? I know Keras Inception V3 doesn't include the auxiliary classifier and also I understood from you implementation that you don't consider the auxiliary classifier.  
Could that be a reason why replicating the JAMA paper is not possible so far? If not, did you manage to replicate the paper? 

thank you and great work.

**Comment by mikevoets** [comment]
Hi, there was no mention of auxiliary classifiers used in the original paper, which is why we haven't considered using this. We assume the most significant reason for why we could not reproduce JAMA's results is the data sets we used for training. They used two data sets from different institutions for training. We used a single data set for training obtained from Kaggle. They have also had a team of ophthalmologists to re-label their data sets, which may have significantly improved the quality of their data sets and therefore are getting better results.
